biogrid/apps/api/src/app/controllers/BiogridController.ts

```diff
@@ -14,23 +12,9 @@ export class BiogridController extends Controller {
     super();
   }
 
-  @SuccessResponse(204)
-  @Post('/')
-  public async NewBiogrid(@Body() body: NewBiogridOpts): Promise<void> {
-    await createNewBiogrid(body);
-  }
-
-  @SuccessResponse(204)
-  @Post('/run')
-  public async RunBiogridSimulation(): Promise<void> {
-    await runBiogridSimulation();
-  }
-
   @SuccessResponse(200)
-  @Get('/simulation-results')
-  public async GetBiogridSimulationResults(): Promise<
-    BiogridSimulationResults
-  > {
-    return await getSimulationResults();
+  @Post('/')
```

Comment:
It seems odd to me to have this "empty" api path.  Even something as simple as "/simulate" would help contextualize